docs/index.md.jinja

```diff
@@ -11,4 +11,5 @@ hidden:
 ---
 
 developer/index
+about/index
```

Comment:
Do we have a rule somewhere about how those files should be named? I named it `index` because it says `developer/index` above. If there is a rule, can you point me to it?

Reply:
Ah, sorry maybe you suggested this because I named my file `about.md`. So should it be `about` or `index`?

Reply:
Since you made it a folder, `index` would make more sense?